css/modals/device-selection/_device-selection.scss

```diff
@@ -36,6 +36,12 @@
                 background-color: rgba(9,30,66,.08);
             }
         }
+        .device-selector-trigger-disabled {
```

Comment:
Will do. "Trigger" is an AtlasKit DropdownMenu term, the thing the user clicks on to open the dropdown. I think trigger might work better in that it's independent of the html element, although "button" is more intuitive right off the bat.

You're right that this is a missing AtlasKit feature, which single-select does have. I'll report to them and add the fixme comments in another commit.